colmena/redis/queue.py

```diff
@@ -246,11 +247,18 @@ def __init__(self, hostname: str, port: int = 6379, name: Optional[str] = None,
         self.value_server_threshold = value_server_threshold
 
         if self.value_server_threshold is not None:
+            if self.serialization_method.lower() != 'pickle':
+                raise ValueError('Serialization method must be pickle to use the value server')
```

Comment:
Is this because the ProxyServer is handling its own serialization internally?

Reply:
Because the `Proxy` objects are not JSON serializable. ProxyStore does handle its own serialization internally but the returned `Proxy` objects still need to be serialized in the `Result` object by Colmena.